### ISSUE TITLE
How to use your code?

### ISSUE THREAD

**Issue by wpchannel** [webgrafia/acf-bidirectional-taxonomy-relation#1]
Hi,

I'd like to use your function but could please provide some explanations? I changed the very last line by my own field but nothing seems to happens. Thanks

**Comment by webgrafia** [comment]
You shoud have an acf relation field in category (or other taxonomy),Put this code in your plugin or in functions.php of your theme, and change 
new Acf_Bidirectional_Tax("category","field_56b9d305d4567) with your taxonomy and acf field id. After saving a relation between category A with category B you should find also B related to A

**Comment by wpchannel** [comment]
Thanks for your quick reply. I've already followed your instructions but is this working with a taxonomy field? I need to link terms between each others for a specific taxonomy. Do you think your code is able to do that?

**Comment by webgrafia** [comment]
Terms are of the same taxonomy?

**Comment by wpchannel** [comment]
Yes. I'd like to link term A with term B in the same taxonomy and make it bidirectionnal.

**Comment by webgrafia** [comment]
it should work.. as I remember... try to debug with some log near update_field function

**Comment by wpchannel** [comment]
Thanks I'll make some tests.
